iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputCommitter.java

```diff
@@ -305,13 +414,14 @@ private Set<FileStatus> listForCommits(JobConf jobConf, String jobLocation) thro
    * Collects the additions to a single table and adds/commits the new files to the Iceberg table.
    * @param io The io to read the forCommit files
    * @param executor The executor used to read the forCommit files
-   * @param jobContext The job context
-   * @param name The name of the table used for loading from the catalog
+   * @param outputTable The table used for loading from the catalog
    * @param location The location of the table used for loading from the catalog
    * @param catalogName The name of the catalog that contains the table
    */
-  private void commitTable(FileIO io, ExecutorService executor, JobContext jobContext, String name, String location,
+  private void commitTable(FileIO io, ExecutorService executor, OutputTable outputTable, String location,
```

Comment:
Would it make sense to put the `location` and the `catalogName` to the `OutputTable` object?

Reply:
Moved `catalogName` into `OutputTable` and `location` can be accessed like
```
outputTable.table.location()
```